pkg/probe/http/http.go

```diff
@@ -135,7 +135,7 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 		return probe.Success, body, nil
 	}
 	klog.V(4).Infof("Probe failed for %s with request headers %v, response body: %v", url.String(), headers, body)
-	return probe.Failure, fmt.Sprintf("HTTP probe failed with statuscode: %d", res.StatusCode), nil
+	return probe.Failure, fmt.Sprintf("HTTP probe failed with statuscode: %d, response body: %v", res.StatusCode, body), nil
```

Comment:
We might want to limit how much of a body we're willing to return here? - A lot of web frameworks will return a bunch of garbage that we probably don't want to shovel around (and wouldn't be useful anyway). - Not sure what the reasonable limit would be though tbh

Reply:
Yeah I am not sure we want to sprintf the entire response body without truncating. Doesn't this also have the potential to leak applications' internal state?

/hold

Reply:
This is moot now, but, for the record, the `body` has already been truncated to `maxRespBodyLength`, a few lines above.